cfme/containers/container.py

```diff
@@ -19,10 +18,7 @@
 
 class ContainerAllView(ContainerObjectAllBaseView):
     """Containers All view"""
-    summary = Text(VersionPick({
-        Version.lowest(): '//h3[normalize-space(.) = {}]'.format(quote('All Containers')),
-        '5.8': '//h1[normalize-space(.) = {}]'.format(quote('Containers'))
-    }))
+    summary = Text('//h1[normalize-space(.) = {}]'.format(quote('Containers')))
```

Comment:
Since this is a constant, can you put the text directly into the locator?

Reply:
@mfalesni just for clearance, do you mean to put it here:
```    def is_displayed(self):
        return Text('//h1[normalize-space(.) = {}]'.format(quote('Containers'))).is_displayed
```

Reply:
```python
summary = Text('//h1[normalize-space(.) = "Containers"]')
```

Reply:
@mfalesni, done.